kolibri/plugins/coach/assets/src/views/home/HomePage/ItemProgressDisplay.vue

```diff
@@ -37,15 +37,15 @@
 <script>
 
   import { validateLinkObject } from 'kolibri.utils.validators';
-  import TextTruncatorCss from 'kolibri.coreVue.components.TextTruncatorCss';
+  import KTextTruncator from 'kolibri-design-system/lib/KTextTruncator';
```

Comment:
Same as my previous comment, I think all imports like this can be removed.

Reply:
I've removed all these imports in the latest commit as well.

Reply:
I tried removing these imports, but it ends up causing a `ReferenceError` in the tests. 

<img width="727" alt="Screenshot 2024-05-31 at 12 52 19" src="https://github.com/learningequality/kolibri/assets/106209849/8bfb63e8-97db-4ca6-b4aa-9c4370a4f98b">
<img width="762" alt="Screenshot 2024-05-31 at 11 58 47" src="https://github.com/learningequality/kolibri/assets/106209849/c14657de-60ef-4d47-a593-02470f8d4dbb">

Reply:
I see, hm I don't understand why it doesn't work, but looking at the Kolibri codebase, it's the case for more components. I will look at it later. So I think there's no need to be dealing with this in the scope of this pull request.

Reply:
@jasonmokk Actually one question - when you removed the `import` statement, did you also remove `KTextTruncator` from `components` on line 48?

Reply:
I did not. I just double checked and still found tests are failing when I remove just that import statement while keeping KTextTruncator in components.

Reply:
>  I just double checked and still found tests are failing when I remove just the import statement while keeping KTextTruncator in components

Can you remove _both_ import statement and `KTextTruncator` from `components` field on line 48?

Reply:
If you don't clean it from `components` and remove import only then `components` don't know where to reach for it and that's most likely the cause of the error. _So both need to be cleaned up_ (and that should be fine because `KTextTruncator` should be globally available for use in all Kolibri components immediately without any extra steps needed). Apologies for my first comment, I wasn't explicit about this and assumed you removed both places. Let me know if this helps.

Reply:
Ah I see. I tried removing `KTextTruncator` from `components` along with the import removal, but unfortunately I am still getting the same `ReferenceError`. Perhaps this is a larger issue?

Reply:
Tests are passing on my localhost after this cleanup. Could you double-check if you removed all places before running tests?

Reply:
I noticed the most recent commit @jasonmokk and thanks for following-up. The tests are still failing, possibly because one last place in the BaseCard, line 77.

Reply:
Got it, my apologies for the delay in the updates as I have been busy with an internship. I have made the final updates, I believe everything should be working now!

Reply:
Got it, I've updated it and I believe everything should be passing now.